lib/src/builders/sound.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:path/path.dart' as p;
+
+class SoundBuilder {
+  List<int> data;
+  String format;
+
+  SoundBuilder({required this.data, required this.format});
+
+  static Future<SoundBuilder> fromFile(File file, {String? format}) async {
+    format ??= p.extension(file.path);
+
+    const formats = {
+      'mp3': 'mpeg',
+      'mpeg': 'mpeg',
+      'ogg': 'ogg',
+    };
```

Comment:
That could be moved as class or "file const"

Reply:
Its only ever used in this function, so it's ok - ImageBuilder does the same